src/components/cv-innhold.tsx

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState } from 'react';
+import { useAppStore } from '../stores/app-store';
+import { ArenaPerson } from '../data/api/datatyper/arenaperson';
+import { UnderOppfolgingData } from '../data/api/datatyper/underOppfolgingData';
+import { LastNedCV } from './cv/last-ned-cv';
+import { RedigerCV } from './cv/rediger-cv';
+import { hentCvOgJobbonsker, hentUnderOppfolging } from '../data/api/fetch';
+import { Heading, Panel } from '@navikt/ds-react';
+import { Errormelding, Laster, Warning } from './felles/minikomponenter';
+import SistEndret from './felles/sist-endret';
+import Sammendrag from './cv/sammendrag';
+import Arbeidserfaring from './cv/arbeidserfaring';
+import Utdanning from './cv/utdanning';
+import AnnenErfaring from './cv/annen-erfaring';
+import Godkjenninger from './cv/godkjenninger';
+import AndreGodkjenninger from './cv/andre-godkjenninger';
+import Kurs from './cv/kurs';
+import Forerkort from './cv/forerkort';
+import Sprak from './cv/sprak';
+import Kompetanser from './cv/kompetanser';
+import Fagdokumentasjoner from './cv/fagdokumentasjoner';
+import './fellesStyling.css';
+
+const CvInnhold = () => {
+    const { fnr } = useAppStore();
+    const [lasterData, setLasterData] = useState<boolean>(true);
+    const [harFeil, setHarFeil] = useState<boolean>(false);
+
+    const [cvOgJobbonsker, setCvOgJobbonsker] = useState<ArenaPerson | null>(null);
+    const [underOppfolging, setUnderOppfolging] = useState<UnderOppfolgingData | null>(null);
+
+    useEffect(() => {
+        const hentCvData = async () => {
+            try {
+                setLasterData(true);
+                const [_cvOgJobbonsker, _underOppfolging] = await Promise.all([
+                    hentCvOgJobbonsker(fnr),
+                    hentUnderOppfolging(fnr)
+                ]);
+
+                setCvOgJobbonsker(_cvOgJobbonsker);
+                setUnderOppfolging(_underOppfolging);
+            } catch (error) {
+                setHarFeil(true);
+            } finally {
+                setLasterData(false);
+            }
+        };
+
+        hentCvData();
+    }, [fnr]);
+
+    if (lasterData) {
+        return (
+            <div tabIndex={2}>
+                <Panel border className="info_panel">
+                    <Laster />
+                </Panel>
+            </div>
```

Comment:
Det er ikke noe i veien for å legge `tabindex` rett på Designsystem-komponentene, så slipper man en ekstra `<div>`!

```suggestion
                <Panel border className="info_panel" tabIndex={2}>
                    <Laster />
                </Panel>
```